src/rinoh/frontend/sphinx/__init__.py

```diff
@@ -288,7 +301,8 @@ def template_from_config(config, confdir, warn):
         except KeyError:
             warn('The language "{}" is not supported by rinohtype.')
 
-    variables = {}
+    variables = config.rinoh_metadata
```

Comment:
Template configuration variables are different from document metadata, so we shouldn't make this assignment. I think this may be caused by some confusion between the terms _Sphinx configuration variables_ and _Template configuration variables_. The corresponding test _test_sphinx_config_rinoh_metadata_ can also be removed.

Reply:
Got it. By this reasoning I would argue that paper_size should not be specified in conf.py since it relates to the layout of the page and should therefore only be specified in the template configuration. (Though this is off topic for the pull request)

Reply:
I strongly agree. See #77.